src/ui/shared/getPopupUrl.ts

```diff
@@ -0,0 +1,9 @@
+import browser from 'webextension-polyfill';
+
+export function getPopupUrl() {
```

Comment:
This function is not used anywhere

Reply:
Ah, alright, that's unfinished refactoring :) I noticed the same pattern to get the default popup URL being used over and over again, so I extracted a function called `getPopupUrl`. I intended to use it but forgot to. Thanks, I'll update other places to call `getPopupUrl` instead of repeating this:
```
  const popupUrl = browser.runtime.getManifest().action?.default_popup;
  if (!popupUrl) {
    throw new Error('popupUrl not found');
  }
  return new URL(browser.runtime.getURL(popupUrl));
```

Reply:
Refactored, now its being used in 3 functions, also `openOnboarding` doesn't take the url parameter anymore (it calls the `getPopupUrl` as well)